data/FilterList.json

```diff
@@ -17106,5 +17106,263 @@
     "name": "HostsY Hosts",
     "syntaxId": 2,
     "viewUrl": "https://raw.githubusercontent.com/Laicure/HostsY_hosts/master/shithosts"
-  }
-]
\ No newline at end of file
+  },
+        {
+            "id": 1608,
+            "name": "Unified Hosts (Little Snitch)",
+            "description": "This repo provide rule groups for Little Snitch based on unified host list to block ads, malware, fake news and porn",
+            "homeUrl": "https://github.com/naveednajam/Little-Snitch---Rule-Groups,
```

Comment:
Missing closing quotes here.

See: https://dev.azure.com/collinbarrett/FilterLists/_build/results?buildId=598&view=ms.vss-test-web.build-test-results-tab&runId=1000096&resultId=100000&paneView=debug

Reply:
Ah, thanks. Should be fixed now.